### PR TITLE
Revoke changes done to Mojang Priority by non-mods/staff

### PR DESCRIPTION
**Pull request by chandler05**
## Purpose
Revoke changes done to the Mojang Priority by those who do not have permissions
#### Checklist
- [ ] Included tests
- [ ] Updated documentation in [README](https://github.com/mojira/arisa-kt/blob/master/README.md) and [wiki](https://github.com/mojira/arisa-kt/wiki)
- [ ] Tested in MCTEST-xxx

### CODE REVIEW

**Comment by chandler05** [comment]
What did I do?
I'm terrible at this :(

File: src/test/kotlin/io/github/mojira/arisa/utils/MockDomain.kt [hunk 1]
```diff
@@ -105,6 +105,7 @@ fun mockIssue(
     updateDescription: (description: String) -> Unit = { Unit },
     updateCHK: () -> Unit = { Unit },
     updateConfirmationStatus: (String) -> Unit = { Unit },
+    updatePriority: (String) -> Unit = { Unit },
```
Comment (SPGoding) [inline src/test/kotlin/io/github/mojira/arisa/utils/MockDomain.kt:108]: This should also be referenced in the body's constructor

Comment (chandler05) [reply src/test/kotlin/io/github/mojira/arisa/utils/MockDomain.kt:108]: thanks, i just saw that :)

**Comment by SPGoding** [comment]
Btw I'm not sure if Arisa has the permission to change Mojang Priority :thinking:

**Comment by chandler05** [comment]
🤦

**Comment by chandler05** [comment]
I updated this pull request, could it be looked at now?

**Comment by chandler05** [comment]
_Actually we have a backlogged PR working on a general RevokeFieldModule: #195_

Also this module doesn't seem to consider #182.

_Originally posted by @SPGoding in https://github.com/mojira/arisa-kt/pull/408#issuecomment-669652906_

**Comment by chandler05** [comment]
As for the backlogged PR, my module could be considered a temporary module in place of that until that one is done, but its fine if my module isn't used 🤷

**Comment by chandler05** [comment]
In fact, maybe this module shouldn't be used at all? I honestly don't care either way

**Comment by chandler05** [comment]
I'm going to close this PR, I don't think its necessary

**Pull request closed without merging**